_config.yml

```diff
@@ -16,7 +16,19 @@ incremental: true
 permalink: pretty
 safe: false
 lsi: false
-exclude: ["_samples", "_scripts", "404.html", "datacenter", "ee", "index.html", "js/metadata.json"]
+
+exclude:
+  - _samples
+  - _scripts
+  - 404.html
+  - datacenter
+  - docker-bake.hcl
+  - ee
+  - index.html
+  - js/metadata.json
+  - LICENSE
```

Comment:
Does this exclude LICENSE from the final image? (wondering if we need to keep it)

Reply:
Yes was not sure if it's relevant to have the LICENSE and README.md on the website actually:
* https://docs.docker.com/LICENSE
* https://docs.docker.com/README.md